test/hotspot/jtreg/vmTestbase/nsk/jvmti/ResourceExhausted/resexhausted001.java

```diff
@@ -62,7 +62,7 @@ public static int run(String args[], PrintStream out) {
                 makeThread();
             }
 
-            System.out.println("Can't reproduce OOME due to a limit on iterations/execution time. Test was useless."
+            System.out.println("Can't reproduce OOME due to a limit on iterations/execution time. resexhausted001 Test was useless."
```

Comment:
```suggestion
            System.out.println("Test resexhausted001: Can't reproduce OOME due to a limit on iterations/execution time. Test was useless."
```
This format reads somewhat better to me.